cassandra/cluster.py

```diff
@@ -3483,6 +3471,8 @@ class PeersQueryType(object):
     _uses_peers_v2 = True
     _tablets_routing_v1 = False
 
+    _resolved_contact_points = None
```

Comment:
It would be better to do the following:
Make `Cluster.endpoints_resolved` a property that returns a composed iterator that does the following:
It iterates over current cached `(Cluster._endpoints_resolved)` resolved endpoints, when it reaches it's end it resolves `self.contact_points` same way it is done in `_resolve_hostnames` stores result in `Cluster._endpoints_resolved` and iterates over it again, skipping endpoints it iterated over before.
And probably has some limit on how often it can reresolve it, say, once a minute